Angular/src/app/app.module.ts

```diff
@@ -0,0 +1,18 @@
+import {NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {DxSchedulerModule, DxPopupModule} from 'devextreme-angular';
+import {AppComponent} from './app.component';
```

Comment:
Let's add brackets to these imports:

```suggestion
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { DxSchedulerModule, DxPopupModule } from 'devextreme-angular';
import { AppComponent } from './app.component';
```